app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderPresenter.kt

```diff
@@ -634,23 +649,12 @@ class ReaderPresenter(
      * manager handles persisting it across process deaths.
      */
     private fun enqueueDeleteReadChapters(chapter: ReaderChapter) {
-        if (!chapter.chapter.read || chapter.pageLoader !is DownloadPageLoader) return
+        if (!chapter.chapter.read) return
```

Comment:
i changed this line from stable.  Cause currently the enqueue to delete would ignore chapters that would have been in a delete position  but had not been just read through the reader.